build.gradle.kts

```diff
@@ -55,7 +55,11 @@ tasks {
 		options {
 			(options as? StandardJavadocDocletOptions)?.apply {
 				encoding = "UTF-8"
-
+				tags(
+					"apiNote:a:API Note:",
+					"implSpec:a:Implementation Requirements:",
+					"implNote:a:Implementation Note:"
+				)
```

Comment:
What's this?

Reply:
This adds support for Javadoc Tags added in Java 8